workflow.md

```diff
@@ -0,0 +1,41 @@
+# Reproducing the study
```

Comment:
Here's the description of the workflow for the paper. Not complete yet. Maybe the plotmaps should get parameters to select the specific borrowing clusters.